electron-builder.yml

```diff
@@ -26,7 +26,7 @@ dmg:
       { x: 122, y: 240, type: file },
     ]
 
-afterSign: ./build-helpers/notarize.js
+# afterSign: ./build-helpers/notarize.js
```

Comment:
this also cannot be commented out - or at least we need to have a `TODO` comment describing next steps when this can be uncommented